administrator/components/com_joomgallery/views/config/tmpl/default.php

```diff
@@ -154,7 +154,7 @@
     JHTML::_('joomconfig.intro', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_THUMBNAILS_INTRO'));
     $directionresize[] = JHTML::_('select.option','0', JText::_('COM_JOOMGALLERY_CONFIG_COMMON_SAMEHIGHT'));
     $directionresize[] = JHTML::_('select.option','1', JText::_('COM_JOOMGALLERY_CONFIG_COMMON_SAMEWIDTH'));
-    $directionresize[] = JHTML::_('select.option','2', JText::_('COM_JOOMGALLERY_CONFIG_COMMON_FREEHEIGHTWIDTH'));
```

Comment:
Unfortunately this change is not backward compatible. You should not change option values so that they cause a different behaviour when e.g. just updating JoomGallery.

Reply:
Thanks! 

@MrMusic: Please would you like to confirm this change being working with some additional selected tests. Thanks in advance.